source/docs/api-reference/api-usage/faults.rst

```diff
@@ -5,8 +5,39 @@ Device Faults
 
 Faults are stored in two fashions. There are "live" faults, which are reported in real-time, and "sticky" faults, which assert persistently and stay asserted until they are manually cleared (like trouble codes in a vehicle).
 
-Sticky Faults can be cleared in Phoenix Tuner X, while a regular fault can only be cleared when the offending problem has been resolved. Faults can be cleared via the :guilabel:`Blink/Clear Faults` button.
+Sticky Faults can be cleared by clicking the :guilabel:`Blink/Clear Faults` button in Phoenix Tuner X, or by calling ``clearStickyFaults()`` on the device in the robot program. A regular fault can only be cleared when the offending problem has been resolved.
 
 .. image:: images/self-test-clear-faults.png
    :width: 70%
    :alt: Blink/Clear faults button in tuner located in the bottom right.
+
+Using API to Retrieve Faults
+----------------------------
+
+Faults can also be retrieved in API using the ``*getFault_*()`` (regular) or ``getStickyFault_*()`` (sticky) functions on the device object. This can be useful for diagnostics or error handling.
```

Comment:
"methods" instead of "functions". We typically use the latter when referring to object methods and the latter for standalone functions.